server/gameconnection.py

```diff
@@ -294,36 +296,54 @@ async def handle_game_result(self, army, result):
         except (KeyError, ValueError):  # pragma: no cover
             self._logger.warning("Invalid result for %s reported: %s", army, result)
 
-    async def handle_operation_complete(self, army, secondary, delta):
-        if not int(army) == 1:
+    async def handle_operation_complete(
+        self, primary: Any, secondary: Any, delta: str
+    ) -> None:
+        """`primary` and `secondary` denote whether or not the primary and
+        secondary mission objectives have been completed"""
```

Comment:
Use the `:param:` notation (see handle_teamkill_happened). In my documentation PR I'm redoing the docstrings to use markdown instead of rst, but just keep it consistent for now. I'll fix it later.